_includes/releases/release-downloads-docker-image.md

```diff
@@ -31,7 +31,12 @@ This release was withdrawn, and we've removed the links to the downloads and Doc
     <a href="https://binaries.cockroachdb.com/cockroach-{{ release.version }}.linux-amd64.tgz"><button id="linux-intel" class="filter-button" data-scope="linux-intel" data-eventcategory="linux-binary-release-notes">Linux Intel</button></a>
     <a onclick="{{ experimental_download_js }}" href="https://binaries.cockroachdb.com/cockroach-{{ release.version }}.linux-3.7.10-gnu-aarch64.tgz"><button id="linux-arm" class="filter-button" data-scope="linux-arm" data-eventcategory="linux-binary-release-notes">Linux ARM<br/>(<b>Experimental</b>)</button></a>
 {% endif %}
-    <a href="https://binaries.cockroachdb.com/cockroach-{{ release.version }}.darwin-10.9-amd64.tgz"><button id="mac" class="filter-button" data-scope="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+{% if release.mac_arm == "false" %}
+    <a href="https://binaries.cockroachdb.com/cockroach-{{ release.version }}.darwin-10.9-amd64.tgz"><button id="mac" class="filter-button" data-scope="mac" data-eventcategory="mac-binary-release-notes">Mac Intel</button></a>
+{% else %}
+    <a href="https://binaries.cockroachdb.com/cockroach-{{ release.version }}.darwin-10.9-amd64.tgz"><button id="mac-intel" class="filter-button" data-scope="mac-intel" data-eventcategory="mac-binary-release-notes">Mac Intel</button></a>
+    <a onclick="{{ experimental_download_js }}" href="https://binaries.cockroachdb.com/cockroach-{{ release.version }}.darwin-11.0-aarch64.unsigned.tgz"><button id="mac-arm" class="filter-button" data-scope="mac-arm" data-eventcategory="mac-binary-release-notes">Mac ARM<br/>(<b>Experimental</b>)</button></a>
```

Comment:
The final tarball name will be `cockroach-{{ release.version }}.darwin-11.0-aarch64.tgz` (without `unsigned`) - we will need to sign the unsigned binaries after the build in order to make them work.